services/master/routes.go

```diff
@@ -55,6 +55,7 @@ func NewService() http.Handler {
 		auth.PUT("/revoke", c.RevokeToken)
 	}
 
+	auth.POST("/gctltoken", m.AuthRequired(), m.LoginHandler)
```

Comment:
```suggestion
	auth.POST("/pat", m.AuthRequired(), m.LoginHandler)
```

change it to pat (personal access token)